doc/releases/squid.rst

```diff
@@ -4,6 +4,375 @@ Squid
 
 Squid is the 19th stable release of Ceph.
 
+v19.2.1 Squid
+=============
+This is the first backport release in the Squid series.
+We recommend that all users update to this release.
+
+Notable Changes
+---------------
+
+* CephFS: Command `fs subvolume create` now allows tagging subvolumes through option
+  `--earmark` with a unique identifier needed for NFS or SMB services. The earmark
+  string for a subvolume is empty by default. To remove an already present earmark,
+  an empty string can be assigned to it. Additionally, commands
+  `ceph fs subvolume earmark set`, `ceph fs subvolume earmark get` and
+  `ceph fs subvolume earmark rm` have been added to set, get and remove earmark from a given subvolume.
+
+* CephFS: Expanded removexattr support for cephfs virtual extended attributes.
+  Previously one had to use setxattr to restore the default in order to "remove".
+  You may now properly use removexattr to remove. You can also now remove layout
+  on root inode, which then will restore layout to default layout.
+
+* RADOS: A performance bottleneck in the balancer mgr module has been fixed.
```

Comment:
s/mgr/Manager/

Reply:
This is widely used language, so I'd prefer to keep it as "mgr".